Templates/templates/CSharp/Requests/MethodRequestBuilder.cs.tt

```diff
@@ -148,5 +120,49 @@ foreach (var m in methods)
 #>
             return request;
         }
+<#
+    // Add request builders for every function bound to the return type of this function.
+    if (composableMethodsInfo != null)
+    {
+        foreach (var m in composableMethodsInfo)
```

Comment:
I'd like to remove the null check to simplify the control logic so we end up with something like:

`foreach (var m in composableMethodsInfo.OrEmptyIfNull())`